src/App.js

```diff
@@ -30,14 +32,27 @@ class App extends Component {
     }
   }
 
-  displayDetails = () => {
+  displayDetails = (event, id) => {
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ movieDetails: data })
```

Comment:
This is where we could've access the movies array from the fetch request.
We could've done `this.setState({ movies: data.movies })`